wordpress-mobile/WordPress-Android
# Epa refactoring on upload error

Refactors "OnUploadError" method in the EditPostActivity. There shouldn't be any changes in the behavior.

To test:
1. Enable "Collect information" in app privacy settings
2. Open Gutenberg
3. Turn on airplane mode
4. Add an image block and try to upload an image
5. Notice the track event in LogCat - something similar to 
`Tracked: editor_upload_media_failed, Properties: {"ext":"jpg","age_ms":685,"bytes":19951,"mime":"image\/jpeg","megapixels":0,"error_type":"CONNECTION_ERROR"}`

- Perform the same test for VideoBlock
- Test both image and video in Aztec


PR submission checklist:
